src/main/java/net/snowflake/ingest/streaming/internal/BlobMetadata.java

```diff
@@ -33,4 +40,9 @@ String getMD5() {
   List<ChunkMetadata> getChunks() {
     return this.chunks;
   }
+
+  @JsonProperty("bdec_version")
+  Integer getVersion() {
+    return bdecVersion;
+  }
```

Comment:
do we need to send this to server side?

Reply:
yes, it is stored in the bdec file format metadata that is used by the scanner to parse the chunk.